FeedSX/screens/postDetail/index.tsx

```diff
@@ -376,15 +429,28 @@ const PostDetail = (props: IProps) => {
 
   // this functions calls the add new reply to a comment api
   const addNewReply = async (postId: string, commentId: string) => {
+    // convert the mentions to route
+    const convertedNewReply = replaceMentionValues(
+      commentToAdd,
+      ({id, name}) => {
+        const PATH = extractPathfromRouteQuery(id);
+        if (!PATH) {
+          return `<<${name}|route://${name}>>`;
+        } else {
+          return `<<${name}|route://${id}>>`;
+        }
+      },
```

Comment:
move this function to a util, as this duplicate function called in various places